conjur-oss/README.md

```diff
@@ -294,7 +294,8 @@ The following table lists the configurable parameters of the Conjur OSS chart an
 
 |Parameter|Description|Default|
 |---------|-----------|-------|
-|`account`|Name of the Conjur account to be used by the Kubernetes authenticator|`"default"`|
+|`account.name`|Name of the Conjur account to be used by the Kubernetes authenticator|`"default"`|
+|`account.create`|Controls wether or not a Conjur account is created automatically after installation|`false`|
```

Comment:
Something that we need added to the README.md are instructions of how to retrieve the admin password when the account is auto-created. (Same content that you've added to the `NOTES` below.)